R/googleComputeEngine.R

```diff
@@ -0,0 +1,23 @@
+#' @export
+#' @importFrom googleComputeEngineR gce_ssh
+docker_cmd.gce_instance <- function(host, cmd = NULL, args = NULL,
```

Comment:
The function method or the importFrom?  I was copying the format of the analogsea method, and the CRAN check said I needed to importFrom. 

But yes, I see no reason it could be in the GCE package